deltachat-ios/Controller/NewChatViewController.swift

```diff
@@ -47,7 +47,7 @@ class NewChatViewController: UITableViewController {
     var hud: ProgressHud?
 
     lazy var deviceContactHandler: DeviceContactsHandler = {
-        let handler = DeviceContactsHandler(dcContext: DcContext())
```

Comment:
wow, this was really a bug. this created new context with new connections, database handles and everything - as a side-effect, this seems to prove, that the same database can be opened twice at the same time :) cc @flub :)

however, instead of using DcContext.shared, we should just use `self.dcContext`

Reply:
:)